core/src/apps/cosmos/transaction.py

```
@@ -117,79 +123,80 @@
+                self.tx = NobleVaultTxn(value, self.msgs_item)
             else:
                 self.tx = UnknownTxn(key, value)
             break
 
         # Separate page
         self.msgs_item.pop("Memo", None)
-        if hasattr(self, "tx") is False:
+        if not hasattr(self, "tx"):
             self.tx = UnknownTxn("Unknown type", "")
 
     def tx_display_make_friendly(self) -> None:
```

Ruff (0.11.9)

`tx_display_make_friendly` is too complex (13 > 10)

(C901)